js/main.js

```diff
@@ -23,4 +31,87 @@ function handleSubmit(event) {
   data.entries.unshift(newObject);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+  $li.prepend(renderTree(newObject));
```

Comment:
You shouldn't prepend to the `li`, you should `prepend` to the `ul`, otherwise your new entry will be an `li` inside of another `li`